tcms/testruns/templates/testruns/mutable.html

```diff
@@ -95,6 +95,59 @@
                 {{ form.build.errors }}
             </div>
         </div>
+        <div class="form-group">
+            <label class="col-md-1 col-lg-1" for="id_start_date">{% trans "Start date" %}</label>
+            <div class="col-md-3 col-lg-3">
+                <div class="input-group date-time-picker-pf">
+                    <input type="text" class="form-control" id="id_start_date">
+                    <span class="input-group-addon">
+                        <span class="fa fa-calendar"></span>
+                    </span>
+                </div>
+
+                {% include "include/datetimepicker_script.html" with selector="#id_start_date" %}
+            </div>
```

Comment:
Let's remove both start & stop dates from this page. The original issue description is incorrect, only the fields for planning must be editable.